sycl/include/CL/sycl/backend.hpp

```diff
@@ -63,15 +64,18 @@ template <backend Backend>
 typename std::enable_if<
     detail::InteropFeatureSupportMap<Backend>::MakePlatform == true,
     platform>::type
-make_platform(const typename interop<Backend, platform>::type &BackendObject) {
+make_platform(
+    const typename backend_traits<Backend>::template input_type<platform>
```

Comment:
shouldn't we replace/remove the `interop` ?

Reply:
I'm not sure removing is a good idea. This seems to be publicly available code, that may have some users. Replacing all instances of `interop` makes sense to me, but I'd prefer to do it as a separate patch, as the change would be quite big.

Reply:
You are right, we can't remove this easily without breaking existing code.